test/config.toml

```diff
@@ -27,6 +32,9 @@ Port = 3498
 NetworkMetricsTick = "43s"
 EthereumMetricsTick = "1m27s"
 
+[maintainer]
+Relay = true
```

Comment:
I would start calling it `BitcoinDifficulty` instead of the `Relay`. I think the `Relay` is pretty cryptic. We've been also using "threshold relay" as an alternative name for the random beacon.

Reply:
@pdyraga 
Should I also change the option for starting from `./keep-client maintainer --relay` to `./keep-client maintainer --bitcoinDifficulty`?